lsp-mode.el

```diff
@@ -7841,10 +7842,11 @@ nil."
                (pcase decompress
                  (:gzip
                   (lsp-gunzip download-path))
-                 (:zip (lsp-unzip download-path (f-parent store-path))))
+                 (:zip (lsp-unzip download-path (f-parent store-path)))
+                 (:targz (lsp-tar-gz-decompress download-path (f-parent store-path)))
```

Comment:
You miss an `)` here. See my propose fix for this PR here https://github.com/kiennq/lsp-mode/commit/cff987c8b1c2746edc8d3fdae81a1aedbc039a72